kairon/api/app/routers/bot/data.py

```
@@ -292,10 +292,21 @@ async def get_collection_data(
+    data = list(DataProcessor.get_collection_data(current_user.get_bot(),
                                                            collection_name=collection_name,
                                                            key=key, value=value, start_idx=start_idx,
-                                                           page_size=page_size))}
+                                                           page_size=page_size))
+    query = {
+        "bot": current_user.get_bot(),
+        "collection_name": collection_name.lower()
+    }
+
+    query.update({
+        f"data__{k}": v for k, v in zip(key, value) if k and v
```

Ruff (0.14.5)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)